api/api.go

```diff
@@ -228,32 +235,21 @@ type connectionStatus struct {
 }
 
 func getMeshStatus(api *API) *meshStatus {
-	if api.mrouter == nil {
+	if api.peer == nil {
 		return nil
 	}
 
-	status := mesh.NewStatus(api.mrouter)
 	strippedStatus := &meshStatus{
-		Name:        status.Name,
-		NickName:    status.NickName,
-		Peers:       make([]peerStatus, len(status.Peers)),
```

Comment:
I think one of these was rendered in the frontend, so we need to make sure that is either removed or replaced with something similar

Reply:
@brancz you're correct. It was added by #1164.